pkgs/development/python-modules/beanstalkc/default.nix

```diff
@@ -1,14 +1,18 @@
-{ stdenv, fetchPypi, buildPythonPackage }:
+{ stdenv, fetchFromGitHub, buildPythonPackage }:
 
 buildPythonPackage rec {
   pname = "beanstalkc";
-  version = "0.4.0";
+  version = "0.5.2";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "98978e57797320146f4b233286d9a02f65d20bad0168424118839fc608085280";
+  src = fetchFromGitHub {
+    owner = "bosondata";
+    repo = "beanstalkc";
+    rev = "v${version}";
+    sha256 = "1dpb1yimp2pfnikmgsb2fr9x6h8riixlsx3xfqphnfvrid49vw5s";
   };
 
+  doCheck = false;
```

Comment:
I tried to run with tests using this: `checkInputs = [ fixtures testfixtures ];` but then got errors. I'm not much of a python guy... Any ideas?

running install tests
running test
running egg_info
writing beanstalkc.egg-info/PKG-INFO
writing dependency_links to beanstalkc.egg-info/dependency_links.txt
writing top-level names to beanstalkc.egg-info/top_level.txt
reading manifest file 'beanstalkc.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*' under directory 'tests'
warning: no previously-included files matching '__pycache__' found under directory '*'
warning: no previously-included files matching '*.py[co]' found under directory '*'
warning: no files found matching '*.rst' under directory 'docs'
warning: no files found matching 'conf.py' under directory 'docs'
warning: no files found matching 'Makefile' under directory 'docs'
warning: no files found matching 'make.bat' under directory 'docs'
writing manifest file 'beanstalkc.egg-info/SOURCES.txt'
running build_ext
no-yaml_fixtures (unittest.loader._FailedTest) ... ERROR

======================================================================
ERROR: no-yaml_fixtures (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: no-yaml_fixtures
Traceback (most recent call last):
  File "/nix/store/yv4pzx3lxk3lscq0pw3hqzs7k4x76xsm-python3-3.7.2/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/build/source/test/no-yaml_fixtures.py", line 1, in <module>
    from fixtures import setup, teardown
ImportError: cannot import name 'setup' from 'fixtures' (/nix/store/ln0606pl00cp7mmx88pks1nj3908y6cx-python3.7-fixtures-3.0.0/lib/python3.7/site-packages/fixtures/__init__.py)


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
error: Test failed: <unittest.runner.TextTestResult run=1 errors=1 failures=0>
builder for '/nix/store/i821602xydczrs8qlq4mgj590rch0s2v-python3.7-beanstalkc-0.5.2.drv' failed with exit code 1
error: build of '/nix/store/i821602xydczrs8qlq4mgj590rch0s2v-python3.7-beanstalkc-0.5.2.drv' failed